src/Eloquent/Model.php

```diff
@@ -251,6 +251,12 @@ public function attributesToArray()
         return $attributes;
     }
 
+    /** @inheritdoc */
+    protected function normalizeCastClassResponse($key, $value)
+    {
+        return [$key => $value];
```

Comment:
At first I was in favor of applying your fix, but it would mean no longer supporting casters that return `[ key => value ]` to change the key name. That would break for people relying on this behavior. For example, if they implement the alternative solution proposed in the other comment.

Reply:
When I was changing it, I was in doubt that it could be a breaking change but I didn't get any error when I ran unit tests.
First, can you give me a sample code for the feature that you mentioned? so I can write tests for that to prevent a breaking change in the future. (I really didn't get what you mean).
Second, when I write tests for that feature, I will figure out this issue.
I think I can solve this issue but first I want to test the feature that you mentioned.